extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/config/config.xml

```diff
@@ -11,9 +11,8 @@
 		</parameter>
 		<parameter name="notificationTimeout" type="integer">
 			<label>Notification Timeout</label>
-			<description>Specifies the amount of time for which the notification sound will be played</description>
-			<default>40</default>
-			<required>true</required>
+			<description>Specifies the amount of time in seconds for which the notification sound will be played</description>
```

Comment:
Because I don't know this ability ;) If it is described somewhere, I will try to change that.